test-network-function/identifiers/identifiers.go

```diff
@@ -461,4 +467,13 @@ the changes for you.`,
 		Remediation: `Ensure that CNF Pod(s) utilize a configuration that supports High Availability.  
 			Additionally, ensure that there are available Nodes in the OpenShift cluster that can be utilized in the event that a host Node fails.`,
 	},
+	TestScalingIdentifier: {
+		Identifier: TestScalingIdentifier,
+		Type:       normativeResult,
+		Description: formDescription(TestScalingIdentifier,
+			`tests that CNF deployments support scale in/out operations. 
+			First, The test starts getting the current replicaCount (N) of the deployment/s with the Pod Under Test. Then, it executes the 
+			scale-in oc command for (N-1) replicas. Lastly, it executes the scale-out oc command, restoring the original replicaCount of the deployment/s.`),
+		Remediation: "",
```

Comment:
I know there is no real useful remediation we can say. But it's probably not good to leave it empty.

Reply:
Done.